rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeperServer.java

```diff
@@ -52,6 +56,7 @@
   protected MetricRegistry metrics;
   protected BookKeeperMetrics bookKeeperMetrics;
   private BookKeeper localBookKeeper;
+  protected static volatile Configuration rubixSiteConfiguration;
```

Comment:
can you please explain why volatile is needed?

Reply:
Statics can be read from thread cache, making volatile to avoid that.